pkgs/development/compilers/rgbds/default.nix

```diff
@@ -0,0 +1,22 @@
+{stdenv, fetchFromGitHub, yacc}:
+
+stdenv.mkDerivation rec {
+  name = "rgbds-${version}";
+  version = "0.2.4";
+  src = fetchFromGitHub {
+    owner = "bentley";
+    repo = "rgbds";
+    rev = "v${version}";
+    sha256 = "0dwq0p9g1lci8sm12a2rfk0g33z2vr75x78zdf1g84djwbz8ipc6";
+  };
+  buildInputs = [ yacc ];
```

Comment:
Is this build-time only? If so perhaps `nativeBuildInputs` makes sense. What do you think?